content/en/docs/architecture/retester.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Retester"
+description: An overview of the retester implementation.
+---
+## What Is Retester?
+Retester is a tool that evaluate which pull requests should be retested, and then retest 
+by [@openshift-ci-robot](https://github.com/openshift-ci-robot) 's commenting `/retest-required` on the pull requests.
+
```

Comment:
Add a description about the old implementation and how it works.

Reply:
{{< alert title="Info" color="info" >}}
The old retester is implemented by a periodic job [periodic-retester](https://github.com/openshift/release/blob/05dd9a1ab5881e55165a0cc0f40d5513e2e2fd11/ci-operator/jobs/infra-periodics.yaml#L260-L300) which uses a GitHub query to filter out the pull requests with failing jobs and a Prow tool [commenter](https://github.com/kubernetes/test-infra/tree/master/robots/commenter) to comment on those pull request. The same query is applied to all GitHub repos and thus it does not recognize any criteria for Prow's auto-merging tool, [Tide](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/tide).
{{< /alert >}}